src/gui/AstroCalcDialog.cpp

```diff
@@ -2159,6 +2174,356 @@ void AstroCalcDialog::saveTransits()
 	}
 }
 
+void AstroCalcDialog::setLunarEclipseHeaderNames()
```

Comment:
Please add the call of this method into retranslate() - this step will solve problem of translation the column names when language is changed.